Libplanet.Tests/Common/Action/DumbAction.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Immutable;
+using System.Threading;
+using Libplanet.Action;
+
+namespace Libplanet.Tests.Common.Action
+{
+    public sealed class DumbAction : IAction
+    {
+        public DumbAction()
+        {
+        }
+
+        public DumbAction(
+            Address targetAddress,
+            string item,
+            bool recordRehearsal = false
+        )
+        {
+            TargetAddress = targetAddress;
+            Item = item;
+            RecordRehearsal = recordRehearsal;
+        }
+
+        public static ThreadLocal<ImmutableList<(Address, string)>>
```

Comment:
(trivial) In the test, asynchronous processing is often performed on `Task` base without threading directly, so it is good to use [`AsyncLocal<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.asynclocal-1?view=netframework-4.7.2).

Reply:
Applied: 7a6ebc034e102fd2f91ec9e83d4e6f0041a744a7.